### PR TITLE
Codejam 3

### PR DESCRIPTION
**Pull request by GeorgyMinsk**

### CODE REVIEW

**Comment by GeorgyMinsk** [comment]
First pull requests

File: src/make.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+function sum(a, b) {
```
Comment (andrei-shelenhouski) [inline src/make.js:1]: @GeorgyMinsk Is it a draft?

File: src/recursion.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+function recursion (tree ){
```
Comment (andrei-shelenhouski) [inline src/recursion.js:1]: @GeorgyMinsk Just a special case

File: src/recursion.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    let massOnLevel=[],massOnLevel2=[];
```
Comment (andrei-shelenhouski) [inline src/recursion.js:2]: @GeorgyMinsk move variables to a different rows

File: src/sumOfOther.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+
```
Comment (andrei-shelenhouski) [inline src/sumOfOther.js:1]: @GeorgyMinsk remove unnecessary rows

File: src/sumOfOther.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+function sumOfOther(mass){
```
Comment (andrei-shelenhouski) [inline src/sumOfOther.js:2]: @GeorgyMinsk Missing space before opening brace

File: src/sumOfOther.js [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+    let resultingArr = [],
```
Comment (andrei-shelenhouski) [inline src/sumOfOther.js:3]: @GeorgyMinsk Split 'let' declarations into multiple statements.

File: src/sumOfOther.js [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+        sumAllElements;
+    sumAllElements=mass.reduce(function(previousValue, currentValue) {
```
Comment (andrei-shelenhouski) [inline src/sumOfOther.js:5]: @GeorgyMinsk Missing space before opening brace.

Comment (andrei-shelenhouski) [reply src/sumOfOther.js:5]: @GeorgyMinsk 'sumAllElements' is never reassigned. Use 'const' instead.

File: src/sumOfOther.js [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+        return previousValue + currentValue;
+      });
+      mass.forEach(function(element) {
```
Comment (andrei-shelenhouski) [inline src/sumOfOther.js:8]: @GeorgyMinsk Expected indentation of 2 spaces

File: src/sumOfOther.js [hunk 9]
```diff
@@ -0,0 +1,13 @@
...
+        resultingArr.push(sumAllElements-element);
+      });
+   return resultingArr
```
Comment (andrei-shelenhouski) [inline src/sumOfOther.js:11]: @GeorgyMinsk missing semicolon

**Review by andrei-shelenhouski — COMMENTED** [review]
@GeorgyMinsk 
sumOfOther = 10
make = 20 * 0
recursion = 20 * 0
git = 20
package.json/linter = 0
tests = 0
all creteria = 0

*30*
=====
FYI: @OlgaMardvilko

**Pull request merged**